site/zenodo_rdm/moderation/models.py

```diff
@@ -73,3 +79,49 @@ def lookup_domain(cls, url):
             .limit(1)
             .scalar()
         )
+
+
+class ModerationQuery(db.Model):
+    """Moderation queries model."""
+
+    __tablename__ = "moderation_queries"
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    """Primary key identifier for the moderation query."""
+
+    score = db.Column(db.Integer, default=0)
+    """Score associated with the query."""
+
+    query_string = db.Column(db.Text, nullable=False)
+    """Query string containing the filter criteria."""
+
+    notes = db.Column(db.Text, nullable=True)
+    """Additional notes or comments regarding the moderation query."""
+
+    active = db.Column(db.Boolean, default=True)
+    """Indicates whether the moderation query is currently active."""
+
+    @classmethod
+    def create(
+        cls, query_string, record_cls=ZenodoRDMRecord, notes=None, score=0, active=True
+    ):
+        """Create a new moderation query with a configurable record class."""
+        query = cls(query_string=query_string, notes=notes, score=score, active=active)
+        db.session.add(query)
+
+        index_percolate_query(record_cls, query_string, active, score, notes)
```

Comment:
Move out, and add a new CLI command to add a single query (so we have the usage)